ecosystem.jenkins.config.js

```diff
@@ -31,14 +31,42 @@ module.exports = {
 			env: {
 				PORT: '9901',
 				SERVICE_BROKER: 'redis://localhost:6379/0',
-				SERVICE_GATEWAY_VOLATILE: 'redis://localhost:6379/3',
-				STRICT_READINESS_CHECK: false,
-				WS_RATE_LIMIT_ENABLE: false,
+				SERVICE_GATEWAY_REDIS_VOLATILE: 'redis://localhost:6379/3',
+				ENABLE_HTTP_API: 'http-status,http-version3,http-exports',
+				ENABLE_WS_API: 'blockchain,rpc-v3',
+				GATEWAY_DEPENDENCIES: 'indexer,connector',
+				WS_RATE_LIMIT_ENABLE: 'false',
 				WS_RATE_LIMIT_CONNECTIONS: 5,
 				WS_RATE_LIMIT_DURATION: 1, // in seconds
-				HTTP_RATE_LIMIT_ENABLE: true,
+				ENABLE_REQUEST_CACHING: 'true',
+				JSON_RPC_STRICT_MODE: 'false',
+				HTTP_RATE_LIMIT_ENABLE: 'true',
 				HTTP_RATE_LIMIT_CONNECTIONS: 200,
 				HTTP_RATE_LIMIT_WINDOW: 10, // in seconds
+				HTTP_CACHE_CONTROL_DIRECTIVES: 'public, max-age=10',
+				ENABLE_HTTP_CACHE_CONTROL: 'true',
+			},
+		},
+		{
+			name: 'lisk-service-blockchain-app-registry',
+			script: 'app.js',
+			cwd: './services/blockchain-app-registry',
+			pid_file: './pids/service_blockchain_app_registry.pid',
+			out_file: './logs/service_blockchain_app_registry.log',
+			error_file: './logs/service_blockchain_app_registry.err',
```

Comment:
just a tiny suggestion. would it be more structured to put these under service directory, removing the `service_` prefix?